frontend/react-neko-chat/src/styles.css

```
@@ -1578,6 +1578,115 @@ button * {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  font-size: 0.9em;
+  line-height: 1.3;
+}
+
+.composer-galgame-options.is-loading .composer-galgame-option {
+  animation: composerGalgameLoadingPulse 1.6s ease-in-out infinite;
+}
+
+@keyframes composerGalgameLoadingPulse {
```

Stylelint (17.9.0)

[error] Expected keyframe name "composerGalgameLoadingPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)